.github/workflows/publish_develop.yml

```diff
@@ -69,3 +69,20 @@ jobs:
           docker push ${{ secrets.AWS_ECR_DEV_REPO }}
           docker tag ${{ secrets.AWS_ECR_DEV_REPO }} ${{ secrets.AWS_ECR_DEV_REPO }}:$TAG
           docker push ${{ secrets.AWS_ECR_DEV_REPO }}:$TAG
+      - name: Push to ECR
+        run: |
+          docker-compose -f docker-compose.dev.yml up --build -d
+          echo 'Displaying containers...'
+          docker ps
+          docker tag covalent-data:latest ${{ secrets.DATA_DEV_REPO }}
+          docker tag covalent-dispatcher:latest ${{ secrets.DISPATCHER_DEV_REPO }}
+          docker tag covalent-queuer:latest ${{ secrets.QUEUER_DEV_REPO }}
+          docker tag covalent-results:latest ${{ secrets.RESULTS_DEV_REPO }}
+          docker tag covalent-runner:latest ${{ secrets.RUNNER_DEV_REPO }}
+          docker tag covalent-ui:latest ${{ secrets.UI_DEV_REPO }}
+          docker push ${{ secrets.DATA_DEV_REPO }}
+          docker push ${{ secrets.DISPATCHER_DEV_REPO }}
+          docker push ${{ secrets.QUEUER_DEV_REPO }}
+          docker push ${{ secrets.RESULTS_DEV_REPO }}
+          docker push ${{ secrets.RUNNER_DEV_REPO }}
+          docker push ${{ secrets.UI_DEV_REPO }}
```

Comment:
should we use secrets for the public ECR registries?

Reply:
That's the standard that we've been using. Obviously the repos themselves are not secret, but we have put all the ECR repos into secrets for consistency.

Reply:
sounds good!